### ISSUE TITLE
[iOS] Camera not close

### ISSUE THREAD

**Issue by Seean-Huang** [juliuscanute/qr_code_scanner#137]
At iOS 14, the status bar has green dot for camera status. When I call controller.dispose, it seems that camera doesn't stop. (The green dot didn't disappear.)

**Comment by ndevev** [comment]
> At iOS 14, the status bar has green dot for camera status. When I call controller.dispose, it seems that camera doesn't stop. (The green dot didn't disappear.)

The same here. Is there any fix?

**Comment by JIAZoro** [comment]
遇到同样的问题，相机好像没关闭

**Comment by chorchannaseang** [comment]
I fixed this issue by adding controller.pauseCamera(); to my statefulWidget dispose method.

@override
void dispose() {
     .............
     controller.pauseCamera();
     controller.dispose();
     super.dispose();
}

**Comment by JERV-code** [comment]
@chorchannaseang  it does not work for me, the camera turns off 1 sec and turn on again after dispose.

did you do something else with controller before dispose ?

**Comment by ndevev** [comment]
> I fixed this issue by adding controller.pauseCamera(); to my statefulWidget dispose method.
> 
> @override
> void dispose() {
> .............
> controller.pauseCamera();
> controller.dispose();
> super.dispose();
> }

It works for me perfectly. Thanks.

**Comment by enricobenedos** [comment]
Hey @Seean-Huang, can you please try with `qr_code_scanner v0.0.14`? It will solve this problem.

**Comment by juliansteenbakker** [comment]
I've tested this on v0.1.0 and i didn't encounter this issue. Please comment if it still persists.

**Comment by enricobenedos** [comment]
Newly reintroduced on `qr_code_scanner: ^0.2.1`, my code has not been changed since `0.0.12`. It was fixed on `0.0.14` and it was working. I tested it on real iOS device.

@juliansteenbakker please tell me if you prefer to open a new issue

```
[✓] Flutter (Channel stable, 1.22.5, on macOS 11.1 20C69 darwin-x64, locale en-IT)
    • Flutter version 1.22.5 at /Users/enrico/SDKs/flutter
    • Framework revision 7891006299 (2 weeks ago), 2020-12-10 11:54:40 -0800
    • Engine revision ae90085a84
    • Dart version 2.10.4

 
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at /Users/enrico/Library/Android/sdk
    • Platform android-30, build-tools 30.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 12.3)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.3, Build version 12C33
    • CocoaPods version 1.10.0

[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin installed
    • Dart plugin version 201.9317
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[✓] VS Code (version 1.52.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.18.0

[✓] Connected device (1 available)
    • iPhone di Enrico (mobile) • 00008020-001C14D20E88002E • ios • iOS 14.3

• No issues found!
```

**Comment by cantomris** [comment]
On iOS, I've got the same problem while using the latest version.
Sacrificing `QrScannerOverlayShape` solved my problem.
There is no issue on Android though.
